### DISCUSSION TITLE
Start ravendb on macos

### DISCUSSION THREAD

**Discussion by djordjedjukic** [ravendb/ravendb#15215] [Q&A] [answered]
Hi, I have some issues with starting ravendb on macOS.
In the past when I run **run.sh** in the terminal I need to "allow access" to every executable file (only the first time).
Now I have an issue where I can not run it at all with the following error (RavenDB-5.4.4-macos-arm64):

`./run.sh: line 30: 69629 Killed: 9               ./Raven.Server --browser`

![image](https://user-images.githubusercontent.com/6117988/199926693-55a78e36-838c-4375-8a00-24e66d48c6d5.png)

Regarding allowing access on macOS, if I may suggest, it would be good if you could do a "Notarization" of the app, then it would be "apple approved", and it could be run without additional work. Here is the link:

https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution

**Comment by ayende** [comment]
Ths is in the process of being hanlded, yes.

**Comment by georgiosd** [comment]
Meantime you can do:  sudo spctl --master-disable

**Comment by djordjedjukic** [comment, marked as answer]
And anything related to this:

`./run.sh: line 30: 89405 Killed: 9               ./Raven.Server --browser`

**Reply by ppekrol** [reply]
RavenDB 5.4.4 got notarized, you would have to re-download the package from the website.

**Reply by djordjedjukic** [reply]
It works now.

Thank you!
